app/src/main/java/com/chartiq/chartiqsample/studies/StudiesActivity.java

```diff
@@ -165,10 +160,21 @@ public void onClick(View v) {
                 for (Study s : lastSelection) {
                     chartIQ.addStudy(s, true);
                 }
-                activeStudies.addAll(lastSelection);
-                studiesAdapter.setActiveStudiesList(activeStudies);
-                studiesAdapter.getAvailableStudies().removeAll(lastSelection);
-                clearSelection();
+                //activeStudies.addAll(lastSelection);
+                //studiesAdapter.setActiveStudiesList(activeStudies);
+                //studiesAdapter.getAvailableStudies().removeAll(lastSelection);
+                //clearSelection();
+
+                chartIQ.getActiveStudies().than(new Promise.Callback<Study[]>() {
```

Comment:
Is `than` proper syntax or a spelling error?

Reply:
Promise is a class that we control, so a typo from when it was first written. There are only a few instances where we use a Promise so it'll be simple enough to change.